databricks.go

```diff
@@ -24,5 +26,5 @@ type Options struct {
 
 var (
 	// DefaultOptions for the driver
-	DefaultOptions = Options{Port: "443", MaxRows: 10000, LogOut: ioutil.Discard}
+	DefaultOptions = Options{Port: "443", ClientId: "go-dbsql", Version: "0.9.0", MaxRows: 10000, LogOut: ioutil.Discard}
```

Comment:
is `DefaultOptions.Version` part of public API for the go sdk? is this settable by the end user?
or is it merely internal implementation and not overwritable by the end user?

If it is part of public API and settable by the end user then we can't protect against end user overwriting the version.

Could you please check if this is a public API of the SDK? or only internal class?

Reply:
DefaultOptions.Version is an internal implementation but in the current implementation, `version=` is an URI argument string that could be added by the end user when opening a connection to override the default. I can remove this override so that end users cannot change the version for now

Reply:
how are you removing the override?

Reply:
we should keep overridable configuration and non-overridable configuration in different constructs.

the naming `DefaultOptions` indicates these are the default values, but the end user may be able to override the defaults.

maybe we should use a different constant variable option to hold version and the clientId rather than reusing `DefaultOptions`. any reason we are using `DefaultOption` for this purpose?

Reply:
Clarification - clientId should be settable by client and I think we could keep this in DefaultOptions? As for version, I agree that if we're not allow clients to override the defaults, then it's best not to use this in DefaultOptions